index.html

```diff
@@ -395,6 +395,28 @@
       </ul>
     </section>
   </section>
+  
+    <section>
+     <h2>IANA Considerations</h2>
+     
+     <p>The link relation type below has been registered by IANA per Section
+  6.2.1 of [[!RFC5988]]:</p>
+     
+     <dl>
+       
+       <dt>Relation Name:</dt>
+       <dd>preload</dd>
+       
+       <dt>Description:</dt>
+       <dd>This link relation type establishes an external resource link that
+       is used to link to a resource and declare its fetch properties.</dd>
```

Comment:
https://github.com/dret/preload/commit/1f21ad208e0b38428e6a4d9a1f028aac98547cd5 fixes the wrong URI.